### PR TITLE
{bio}[foss/2017b,intel/2017b] RELION v2.1

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.7, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/7c75cf40101f4ef272e57a63800dba24 for a full test report.

File: easybuild/easyconfigs/r/RELION/RELION-2.1-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+name = 'RELION'
+version = '2.1'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/r/RELION/RELION-2.1-foss-2017b.eb]: You want toolchainopts = {'openmp': True} for both of them.

Comment (vanzod) [reply easybuild/easyconfigs/r/RELION/RELION-2.1-foss-2017b.eb]: Good point here. I noticed that that toolchain flag is not present in any `RELION` easyconfig. Maybe worth considering opening another PR that fixes this?

Comment (vanzod) [reply easybuild/easyconfigs/r/RELION/RELION-2.1-foss-2017b.eb]: One question. How does the addition of `-fopenmp` interacts with `-lpthread` in this context?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.7, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/2d00850a86ce10ff10aa437e775b0098 for a full test report.

File: easybuild/easyconfigs/r/RELION/RELION-2.1-foss-2017b.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/r/RELION/RELION-2.1-foss-2017b.eb]: As far as I have seen there is no problem.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2077.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/357e3e7c1bc19973791db0af9af861b2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (2 easyconfigs in this PR)
node3134.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/15fd56b3d060d49338fde3393ebd7829 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/39f0e073a6d4d9415b1cadccb05f1411 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**